_data/guides.yaml

```diff
@@ -31,6 +31,9 @@ categories:
       - title: Securing WildFly Apps with SAML on OpenShift
         url: /guides/security-saml-openshift
         description: Learn how to secure applications deployed to WildFly on OpenShift with SAML.
+      - title: Securing WildFly Apps with Okta on OpenShift
+        url: /guides/security-oidc-okta-openshift
+        description: Learn how to secure applications deployed to WildFly on OpenShift with the Okta OpenID provider.
```

Comment:
@fjuma (and @PrarthonaPaul / @hpehl)

How do you want these security ones organized? Looking at https://www.wildfly.org/guides/ they look to be output in the order here, going left to right, with a new row after four. So is putting this 2nd on the 2nd row what we want? Or somewhere else? Any other reordering we should do?

Related, what about the titles (and perhaps the description)? Any tweaks we should do not that we have a bunch? Particularly 'Securing WildFly Apps with OpenID Connect on OpenShift' which seems like a 'Keycloak' element of the 'Securing WildFly Apps with [Auth0|Okta] on OpenShift' set, but has a differently phrased title/description.

Note that my comment here is out-of-scope for this PR; i.e. if you're fine Farah with what's been done for this guide we can merge it and if there's anything to be done re my comments here that can be done separately.

Reply:
I think it would be good to reorder things so that the guides about the various OpenID providers are one after the other with consistent titles/descriptions. I'll submit a follow-up PR for that, we can merge this guide as is. Thanks very much @bstansberry for noticing it.